sorc/checkout.sh

```diff
@@ -7,9 +7,10 @@ echo $topdir
 echo fv3gfs checkout ...
 if [[ ! -d fv3gfs.fd ]] ; then
     rm -f ${topdir}/checkout-fv3gfs.log
-    git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
+#    git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
+    git clone https://github.com/junwang-noaa/ufs-weather-model fv3gfs.fd
     cd fv3gfs.fd
-    git checkout GFS.v16.0.1
+    git checkout esmf8.0.1
```

Comment:
Kate added Externals.cfg to the workflow.   This also needs to be updated to get the correct snapshot of the model.

Reply:
Waiting for new FV3 tag, will update when available.